tilings/strategies/unfusion.py

```diff
@@ -0,0 +1,317 @@
+from collections import Counter
```

Comment:
Same thing here. We unfuse either every row, or every column to make sure that no extra assumptions are needed. We can discuss later whether to unfuse a single row, or single column, but then assumptions are needed.

This strategy will be applied to every tiling so make sure to use it as an expansion strategy. Will discuss later (agenda) whether to give it an option to be only applied to "small" tilings